### PR TITLE
Add base for custom web app

### PR DESCRIPTION
**Pull request by msmolens**
This replaces the current Girder web client extensions with a minimal custom web app. The app has a header bar, a front page, an example view, and now a mockup for an upload dataset view.

### CODE REVIEW

**Comment by brianhelba** [comment]
Let's change the namespace to `isic`.

**Comment by brianhelba** [comment]
Drop the upload dataset view functionality from this PR please. We'll do that separately.

File: server/__init__.py [hunk 1]
```diff
@@ -98,3 +131,8 @@ class Root(object):
     info['apiRoot'].image = api.ImageResource()
     info['apiRoot'].segmentation = api.SegmentationResource()
     info['apiRoot'].study = api.StudyResource()
+
+    # Move girder app to /girder, serve isic_archive app from /
+    info['serverRoot'], info['serverRoot'].girder = (Webroot(),
```
Comment (brianhelba) [inline server/__init__.py]: Let's keep the Girder app mounted at `/`. Mount this new app at `/isic` for now.

File: server/__init__.py [hunk 2]
```diff
@@ -98,3 +131,8 @@ class Root(object):
...
+                                                     info['serverRoot'])
+    info['serverRoot'].api = info['serverRoot'].girder.api
```
Comment (brianhelba) [inline server/__init__.py]: We're not ready to remap the API yet, since we still have Angular code where it's hardcoded. Comment out this line.

**Comment by msmolens** [comment]
@brianhelba PTAL

**Comment by brianhelba** [comment]
LGTM.

**Pull request merged**